### PR TITLE
Add tests for trace

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
```
(virtualenv)/m/s/s/st2 git:master ❯❯❯ st2 run tests.test_quickstart_trace                                                ✭ ◼
............
id: 55d7b4a832ed35048e764a7f
action.ref: tests.test_quickstart_trace
status: succeeded
start_timestamp: 2015-08-21T23:30:48.097556Z
end_timestamp: 2015-08-21T23:31:10.851891Z
+--------------------------+-----------+---------------------+---------------------+---------------------+
| id                       | status    | task                | action              | start_timestamp     |
+--------------------------+-----------+---------------------+---------------------+---------------------+
| 55d7b4a832ed350493372ab7 | succeeded | test_trace_tag_duri | core.local          | Fri, 21 Aug 2015    |
|                          |           | ng_execution        |                     | 23:30:48 UTC        |
| 55d7b4ab32ed350493372abb | succeeded | sleep               | core.local          | Fri, 21 Aug 2015    |
|                          |           |                     |                     | 23:30:51 UTC        |
| 55d7b4b632ed350493372abe | succeeded | test_trace_list     | core.local          | Fri, 21 Aug 2015    |
|                          |           |                     |                     | 23:31:02 UTC        |
| 55d7b4b732ed350493372ac1 | succeeded | get_trace_id_for_ta | core.local          | Fri, 21 Aug 2015    |
|                          |           | g                   |                     | 23:31:03 UTC        |
| 55d7b4b832ed350493372ac4 | succeeded | test_trace_id_durin | core.local          | Fri, 21 Aug 2015    |
|                          |           | g_execution         |                     | 23:31:04 UTC        |
| 55d7b4bb32ed350493372ac8 | succeeded | grep_trace_id_execu | core.local          | Fri, 21 Aug 2015    |
|                          |           | tions               |                     | 23:31:07 UTC        |
| 55d7b4bd32ed350493372acb | succeeded | assert_trace_id_has | asserts.object_key_ | Fri, 21 Aug 2015    |
|                          |           | _two_executions     | string_equals       | 23:31:09 UTC        |
+--------------------------+-----------+---------------------+---------------------+---------------------+
(virtualenv)/m/s/s/st2 git:master ❯❯❯
```

### CODE REVIEW

File: packs/tests/actions/chains/test_quickstart_trace.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    -
+      name: "test_trace_tag_during_execution"
+      ref: "core.local"
+      params:
+        env:
+          ST2_AUTH_TOKEN: "{{token}}"
+        cmd: "st2 run core.local cmd=date --trace-tag='test-trace-tag'"
+      on-success: sleep
+    -
+      name: "sleep"
```
Comment (manasdk) [inline packs/tests/actions/chains/test_quickstart_trace.yaml]: why is the sleep needed?

**Comment by manasdk** [comment]
Awesome! I think.

File: packs/tests/actions/chains/test_quickstart_trace.yaml (hunk 1 above)
Comment (lakshmi-kannan) [inline packs/tests/actions/chains/test_quickstart_trace.yaml]: Give some time for execution to run.

Comment (manasdk) [inline packs/tests/actions/chains/test_quickstart_trace.yaml]: st2 run will not return until the execution is complete. You are not using -a

**Pull request merged**